modules/azure/billing-alert-functions/subscription-budgets/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.96.0"
+    }
+  }
+}
+
+resource "azurerm_monitor_action_group" "slack" {
+  count               = var.enable_slack ? 1 : 0
+  name                = "action-group-slack-${var.func_identifier}"
```

Comment:
according to: https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations we will want to name this ag-${var.func_identifier}-${var.env}-${var.location}